lib/gui-utils-tft/src/TFTUtils.cpp

```diff
@@ -431,6 +433,12 @@ void TFTUtils::suspend() {
     esp_bluedroid_disable();
     esp_bt_controller_disable();
     esp_wifi_stop();
+
+    rtc_gpio_init(GPIO_NUM_14);
+    rtc_gpio_set_direction(GPIO_NUM_14, RTC_GPIO_MODE_OUTPUT_ONLY);
+	rtc_gpio_set_level(GPIO_NUM_14, 1);
```

Comment:
please change GPIO_NUM_14 to ADC_EN (hal.h define)

Reply:
Don´t admit this changue, only GPIO_NUM_14

Reply:
ok